File: 01_objective_c/03/05/06/07/08/09/10/11/12/13/14/15/15_attribuesDesPropriete/15_attribuesDesPropriete/ViewController.m
//
//  ViewController.m
//  15_attribuesDesPropriete
//
//  Created by etudiant on 4/18/17.
//  Copyright © 2017 etudiant. All rights reserved.
//

#import "ViewController.h"
#import "Societe.h"
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    Societe * societe = [[Societe alloc]init];
    [societe ecrireNom:@"Ma société"]; //Rappel: on a changer le nom du setter
    NSLog(@"Le nom de la société : %@",[societe lireNom]);
    
    [societe configurerSociete];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: 04_devApplication_IOS_avec_Swift/05_Passage d'un ecran à l'autre/05_Passage d'un ecran à l'autre/ViewController.swift
//
//  ViewController.swift
//  05_Passage d'un ecran à l'autre
//
//  Created by etudiant on 5/11/17.
//  Copyright © 2017 etudiant. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func btnDestinationSansSegue(_ sender: UIButton) {
        //creer instance du controlleur de destination
        
        //idDestinationSansSegue est l'id defini dans le main.storyboard pour le controleur destinationSansSegue, dans le champ 'storyboard id' par l'assistant d'identification
        let controleurDestination =
            storyboard?.instantiateViewController(withIdentifier: "idDestinationSansSegue")
                as! DestinationSansSegueViewController
        
        //afficher ce controller
        present(controleurDestination, animated: true, completion: nil)
    }

}

